src/instrument/callbacks/spec_data_file_writer.py

```diff
@@ -8,22 +8,17 @@
     "newSpecFile",
 ]
 
-import logging
-
-logger = logging.getLogger(__name__)
-logger.info(__file__)
-
-try:
-    import apstools.callbacks as APS_fw
-except ModuleNotFoundError:
-    import apstools.filewriters as APS_fw
-
 import datetime
+import logging
 import pathlib
 
+import apstools.callbacks as APS_fw
 import apstools.utils
 
-from ..framework.initialize import RE
+from ..initialize_bs_tools import RE
```

Comment:
Move this _after_ the `logger.info()` report.  It is a useful diagnostic when used in that order.

Reply:
Probably this comment applies with respect to any local imports.  Should report the current module name first, then local imports.  Shows the order of modules imported.

Reply:
It's application of a _lesson learned_.